### PR TITLE
Fixed compiler errors in monodevelop_instance.cpp

### PR DESCRIPTION
**Pull request by BrainBlasted**
There were a couple compiler errors in this file, can be viewed [here](https://hastebin.com/qomugoseki.js).

### CODE REVIEW

File: mono/editor/monodevelop_instance.cpp [hunk 1]
```diff
@@ -36,7 +36,7 @@ void MonoDevelopInstance::execute(const Vector<String> &p_files) {
 	MonoObject *exc = NULL;
 
 	Variant files = p_files;
-	const Variant *args[1]{ &files };
+	const Variant *args[1];{ &files; };
```
Comment (vnen) [inline mono/editor/monodevelop_instance.cpp]: That does not make sense. I assume it was supposed to be `const Variant *args[1] = { &files };` or something like it.

Comment (karroffel) [reply mono/editor/monodevelop_instance.cpp]: Haha, yes this effectively does this

```cpp
const Variant *args[1]; // hey cool, it's an array
// let's open a scope
{
    &files; // well, this doesn't really do anything.
}
```

I love C/C++ grammar :)

@vnen's fix is correct.

Comment (BrainBlasted) [reply mono/editor/monodevelop_instance.cpp]: Made the changes requested.

**Review by karroffel — APPROVED** [review]

**Comment by neikeq** [comment]
What compiler were you using? I can't believe neither gcc or msvc errored with this.

**Comment by karroffel** [comment]
@neikeq maybe it's used as an initializer list, in which case it's perfectly fine.

**Comment by BrainBlasted** [comment]
I was using Clang

**Comment by neikeq** [comment]
Ah, right. I'm using C++11. Any idea how to tell scons to disable it? I'm tired of not noticing these errors until travis notifies me.

**Comment by akien-mga** [comment]
I guess `-std=c++03`?

**Comment by neikeq** [comment]
But how do you pass compiler flags to scons?

**Comment by akien-mga** [comment]
It reads them from the environment AFAIK. I think there's also a scons option to define them, but env works. Not sure if `CCFLAGS` works though or if it should be `CFLAGS`+`CXXFLAGS`.
```
export CCFLAGS='-std=c++03'
scons p=x11
```

**Pull request merged**